nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenExpr.scala

```diff
@@ -1277,23 +1278,26 @@ trait NirGenExpr { self: NirGenPhase =>
                               _))))))),
             _),
             _) =>
-          val chars = Val.Chars(str.replace("\\n", "\n").replace("\\r", "\r"))
-          val const = Val.Const(chars)
-          buf.box(nir.Rt.BoxedPtr, const, unwind)
+        // format: on
+        val chars = Val.Chars(StringUtils.processEscapes(str))
```

Comment:
The indentation is wrong here.

In general, there are zillions of formatting changes that are unrelated to this PR in this file. They could have come from a transient use of another version of scalafmt or some misconfiguration. Could you please revert all the changes in this file, then reintroduce this specific change (which is only 1 line)?

Reply:
Sure, it should be fixed now